server.go

```diff
@@ -1372,6 +1417,24 @@ func (sp *serverPeer) OnAddr(p *peer.Peer, msg *wire.MsgAddr) {
 // OnRead is invoked when a peer receives a message and it is used to update
 // the bytes received by the server.
 func (sp *serverPeer) OnRead(p *peer.Peer, bytesRead int, msg wire.Message, err error) {
+	// Ban peers sending messages that do not conform to the wire protocol.
+	switch tErr := err.(type) {
```

Comment:
Noted, I'm curious to know what the default value for `wire.ErrorCode` is going to be, wondering if it will default to an error, if not what will be the default if not zero.